packages/icons-angular/package.json

```diff
@@ -1,16 +1,19 @@
 {
+  "$schema": "./node_modules/ng-packagr/package.schema.json",
   "name": "@carbon/icons-angular",
   "private": true,
   "description": "Angular components for icons in digital and software products using the Carbon Design System",
   "version": "10.6.0",
   "license": "Apache-2.0",
   "main": "lib/index.js",
+  "ngPackage": {
+    "lib": {
+      "entryFile": "ts/index.ts"
+    },
+    "whitelistedNonPeerDependencies": ["@carbon/icon-helpers"]
+  },
   "repository": "https://github.com/carbon-design-system/carbon/tree/master/packages/icons-angular",
   "bugs": "https://github.com/carbon-design-system/carbon/issues",
-  "files": [
-    "lib",
```

Comment:
Is `files` covered by somewhere else?

Reply:
Yep 👍 `ng-packagr` handles generating the final `package.json`